File: cpp/bmi.cxx
/*
 * bmi.cxx

 * 
 */


#include <iostream>
using namespace std;
#include <cmath>
int main(int argc, char **argv)
{
    float masa ; 
	cout << " Podaj masę:";
    cin >> masa;
    
    float wzrost ; 
	cout << " Podaj wzrost:";
    cin >> wzrost;
    
    float bmi ;
    
    bmi = masa/(wzrost*wzrost);

    cout<< "BMI = "<< bmi <<endl;
    
    if (bmi < 18.5)
    {  
        cout<<"Niedowaga"<<endl;
    }
    else if (bmi >= 18.5 && bmi <25)   
        cout<<"Norma"<<endl;
    else if (bmi >= 24.9 && bmi <30)
        cout<<"Nadwaga"<<endl;
    else
        cout<<"Otyłość"<<endl;
	
    
	return 0;
}

File: Kurs/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  bmi.py
#   


def main(args):
    
    masa = float(input('Masa : '))
    print(masa)
    
    wzrost = float(input('Wzrost: '))
    print(wzrost)
    
    bmi = masa/((wzrost*0.01)**2)
    
    print ('Twoje BMI = ', bmi)
    if bmi < 18.5:
        print("Niedowaga")
    
    elif bmi <25:   
        print("Norma")
        
    elif bmi  <30:
        print("Nadwaga")
    
    elif bmi > 30:
       print("Otyłość") 
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
